src/tests/user.rs

```diff
@@ -66,12 +66,18 @@ fn user_insert() {
 #[test]
 fn me() {
     let (_b, app, middle) = ::app();
-    let mut req = ::req(app, Method::Get, "/me");
+    let mut req = ::req(app.clone(), Method::Get, "/me");
     let response = t_resp!(middle.call(&mut req));
     assert_eq!(response.status.0, 403);
 
-    let user = ::mock_user(&mut req, ::user("foo"));
-
+    // with GET /me update gives 404 response
+    // let user = ::mock_user(&mut req, ::user("foo"));
```

Comment:
I'm not sure I understand the difference, could you clarify? I forgot to delete these commented out lines, what I was getting at is that `mock_user()` seems to do the same thing as calling `new_user()` and `sign_in_as(user)` used below except below doesn't cause the 404.